CodeIgniter/application/config/config.php

```diff
@@ -23,7 +23,7 @@
 | a PHP script and you can easily do that on your own.
 |
 */
-$config['base_url'] = 'http://benoats.co/university/web-enterprise-system/codeigniter';
+$config['base_url'] = 'http://localhost:8888/CodeIgniter/';
```

Comment:
would this not need to be http://benoats.co/university/web-enterprise-system/codeigniter